spec/controllers/hub/organizations_controller_spec.rb

```diff
@@ -22,7 +22,7 @@
   describe "#create" do
     let(:params) do
       {
-        organization: {
+        hub_organization_form: {
```

Comment:
this doesn't reflect the fact that there are two new params being passed in but also the controller spec and the form spec test the same thing which is what happens to the records. the controller could instead mock the form and the form spec could be the only one to test the outcome for the records.